### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "build:css": "npm run watch:css -- --no-watch",
     "watch:js": "webpack -w -h",
     "watch:css": "mkdirp public/css && autoless less public/css --autoprefix '> 1%, last 2 versions'  --source-map",
-    "server": "node server.js"
+    "server": "node server.js",
+    "test": "npm-run-all --parallel start test:selenium",
```
Comment (alicoding) [inline package.json]: I think we will need a way to close the server somehow too?

**Comment by alicoding** [comment]
Your test is failing and I left one comment there.

File: package.json (hunk 1 above)
Comment (ScottDowne) [inline package.json]: Yeah, agreed, but I'm not sure how yet.

**Comment by ScottDowne** [comment]
Odd, why is Travis running the tests? I didn't tell it to do that yet and no idea if it would event work.

**Comment by alicoding** [comment]
r+ but file to get this run by travis.

**Pull request merged**